pkgs/applications/science/math/pari/default.nix

```diff
@@ -15,10 +17,10 @@ stdenv.mkDerivation rec {
 
```

Comment:
You can probably make libpthreadstubs optional as well

Reply:
You're right